src/components/elements/side-tab.tsx

```diff
@@ -4,7 +4,7 @@ import { Text, Header } from './typography'
 import device, { size } from 'themes/device'
 import { getWindowWidth } from 'common/utility'
 import { Box } from 'components/containers'
-import { Desktop } from 'components/containers/show'
+import { Desktop } from 'components/containers/visibility'
```

Comment:
This can be imported directly from `components/containers`

Reply:
ok